aws/image/files/etc/systemd/system/kata-agent.service

```diff
@@ -5,6 +5,11 @@ After=network.target
 [Service]
 ExecStart=/usr/local/bin/kata-agent --config /etc/agent-config.toml
 ExecStartPre=-umount /sys/fs/cgroup/misc
+# Download and unpack container images on the root disk
+ExecStartPre=-mkdir -p /image /run/image
+ExecStartPre=-+mount --bind /image /run/image
+ExecStartPre=-mkdir -p /kata-containers /run/kata-containers
+ExecStartPre=-+mount --bind /kata-containers /run/kata-containers
 ExecStartPre=ip netns add podns
 ExecStartPre=ip netns exec podns ip link set lo up
 ExecStopPost=ip netns delete podns
```

Comment:
I think unmounitng the directories is necessary when the service is stopped.
```
ExecStopPost=umount /image
ExecStopPost=umount /kata-containers
```

Reply:
Thanks, I'll add that in :)

Reply:
Should be sorted now.